docs/content/4.sdk/2.getting-started/1.index.md

```diff
@@ -72,7 +72,15 @@ Let's break down the code above:
 
 - The `middlewareUrl` is the URL of the middleware instance.
 
-- The `getRequestHeaders` function is used to retrieve the Cookie header with cookie values.
+- The `getRequestHeaders` function is used to provide the incoming headers within your requests. You can use `getRequestHeaders` to access and proxy the initial cookie headers to SDK requests during SSR. Initial headers could be provided by the [`getSdk`](#getsdk) method.
+  Check out examples there:
+    - Next Pages Router [link](https://github.com/vuestorefront/vue-storefront/tree/main/packages/storefront/packages/next/__tests__/apps/pages-router)
+    - Next App Router [link](https://github.com/vuestorefront/vue-storefront/tree/main/packages/storefront/packages/next/__tests__/apps/app-router)
+    - Nuxt app [link](https://github.com/vuestorefront/vue-storefront/tree/main/packages/storefront/packages/nuxt/__tests__/app)
+
+::info
+In the browser, `getRequestHeaders` will return an empty object.
```

Comment:
In unified, we have
<img width="1105" alt="image" src="https://github.com/vuestorefront/vue-storefront/assets/45824492/ad452c5f-b1ea-4af9-a59b-0cde416649af">

One of the questions I've seen come up is: "How do I use `getRequestHeaders` with `useSdk`? 
Can you please address this? Is it possible? Is it recommended? I see the comment now that says "In the browser, `getRequestHeaders` will return an empty object." Can we take this one step further and explain usage with `useSdk`, or if it's not recommended, why and what to do instead if a dev feels it's needed. 🙂 Thanks in advance